File: idea-plugin/src/main/java/ru/mipt/acsl/decode/idea/plugin/DecodeGenerateSqliteForDecodeSourcesAction.java
package ru.mipt.acsl.decode.idea.plugin;

import com.intellij.openapi.actionSystem.AnAction;
import com.intellij.openapi.actionSystem.AnActionEvent;
import org.apache.commons.lang.NotImplementedException;

/**
 * @author Artem Shein
 */
public class DecodeGenerateSqliteForDecodeSourcesAction extends AnAction
{

    public static final String GROUP_DISPLAY_ID = "Decode SQLite Generation";

    @Override
    public void actionPerformed(AnActionEvent anActionEvent)
    {
        throw new NotImplementedException();
    }
}

File: src/main/scala/ru/mipt/acsl/decode/idea/plugin/DecodeGenerateSqliteForDecodeSourcesAction.scala
package ru.mipt.acsl.decode.idea.plugin

import com.intellij.openapi.actionSystem.AnAction
import com.intellij.openapi.actionSystem.AnActionEvent
import org.apache.commons.lang.NotImplementedException

/**
 * @author Artem Shein
 */
class DecodeGenerateSqliteForDecodeSourcesAction extends AnAction {

    val GROUP_DISPLAY_ID = "Decode SQLite Generation"

    override def actionPerformed(anActionEvent: AnActionEvent): Unit = throw new NotImplementedException
}
